packages/block-library/src/table/style.scss

```diff
@@ -17,25 +28,29 @@
 		// The table element needs to be kept as display table
 		// for table features to work reliably.
 		display: table;
-
 		// Table cannot be 100% width if it is aligned, so set
 		// width as auto.
 		width: auto;
+
+		td,
+		th {
+			word-break: break-all;
```

Comment:
Why is this needed? Does it deserve a comment?

Reply:
Assuming we don't want tables to horizontally scroll when they are aligned, we need table content to break. Added comment.